packages/common/src/services/file-upload.service.ts

```diff
@@ -8,24 +8,22 @@ import { STORAGE_MODE } from '../interfaces/file-upload.interface';
 
 export class FileUploadService {
   private readonly storage: any;
-  private readonly useMinio: boolean;
+  private readonly useMinio: boolean = process.env.STORAGE_MODE?.toLowerCase() === STORAGE_MODE.MINIO;
```

Comment:
use config service instead of `process.env`

```suggestion
configService.get<string>('STORAGE_MODE');
```